website/pages/docs/internals/security.mdx

```diff
@@ -0,0 +1,322 @@
+    built-in ACL system for fine-grained control over jobs.
+
+* **[Namespaces](/guides/governance-and-policy/namespaces.html)**
+    (**Enterprise Only**) - This feature allows for a cluster to be shared by
+    multiple teams within a company. Using this logical separation is important
+    for multi-tenant clusters to prevent users without access to that namespace
+    from conflicting with each other. This requires ACLs to be enabled in order
+    to be enforced.
+
+* **[Resource Quotas](/guides/governance-and-policy/quotas.html)**
+    (**Enterprise Only**) - Can limit a namespace’s access to the underlying
+    compute resources in the cluster by setting upper-limits for operators.
+    Access to these resource quotas can be managed via ACLs to ensure read-only
+    access for operators so they can’t just change their quotas.
+
+#### Recommendations
+
+The following are security recommendations that can help significantly improve
+the security of your cluster depending on your use case. We recommend always
+practicing defense in depth when architecting the security mechanisms for your
+environment.
+
+* **[Rotate Credentials](/docs/job-specification/vault.html)** -
+    Using something like [Vault](/docs/vault-integration/index.html) to
+    create and manage dynamic, rotated credentials is highly recommended to
+    prevent secrets from being easily exposed within the [job
+    specification](/docs/job-specification/index.html)
+    itself which may be leaked into version control or otherwise be accidently
+    stored on disk on an operator’s local machine. It is also possible to
+    [integrate with Vault’s PKI secret engine](/guides/security/vault-pki-integration.html)
+    to automatically generate and renew dynamic, unique X.509 certificates for
+    each Nomad node with a short
+    [TTL](https://en.wikipedia.org/wiki/Time_to_live).
+
+* **[Running without Root](https://groups.google.com/forum/#!topic/nomad-tool/pSyMwC_FSFA)** -
+    Certain features of Nomad can be used without needing to run the Nomad agent
+    server or client as the `root` user. Instead you can granularly assign the
+    appropriate capabilities in various ways for your Nomad agents. For example:
+    Nomad servers only require access to the data directory; it is possible to
+    use Nomad to orchestrate Docker containers by adding a non-root `nomad` user
+    to the `docker` group to access the [default unix
+    socket](https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-socket-option).
+
+* **Containers with Sandbox Runtimes** - In some situations, such as running
+    untrusted code as a service, it may be worth considering using different
+    container runtimes such as [gVisor](https://gvisor.dev/) or [Kata
+    Containers](https://katacontainers.io/). These types of runtimes provide
+    sandboxing features which help prevent raw access to the underlying shared
+    kernel for other containers and the Nomad client agent itself.
```

Comment:
We should include a link to how to do that: https://www.nomadproject.io/docs/drivers/docker#runtime